### ISSUE TITLE
Sprint particle physics: off is ignored [BUG]

### ISSUE THREAD

**Issue by JasonHorkles** [haubna/PhysicsMod#663]
**Describe the bug**
Turning off sprint particle physics still has blocky particles when you run

**To Reproduce**
Set `Sprint particle physics` to off and sprint

**Screenshots**
![image](https://github.com/haubna/PhysicsMod/assets/51760372/f41626a4-6a3d-433d-baf8-5c4e3c3651cc)

Minecraft Version: 1.19.4
Physics Mod Version: 2.12.3

**Comment by haubna** [comment]
Yeah that's the downside of having the block particle physics option. The sprint particles are still "block particles" (particles that use block textures) so they will stay on.

**Comment by JasonHorkles** [comment]
So I just turned the block particle physics setting off and it looks like it did fix that and also fixed the issue with barriers. Is it possible to exclude barrier particles from that option when enabled? Physics type is set to off on barriers but they still seem to have the physics particles when block particle physics is on. I do honestly like the block particles when I jump down and whatnot so it would be nice if I could have it on without the barriers going wack

**Comment by haubna** [comment]
What's the problem with barriers?

**Comment by JasonHorkles** [comment]
> What's the problem with barriers?

When block particle physics is turned on it makes barriers almost impossible to see
![image](https://user-images.githubusercontent.com/51760372/235324084-ab233a40-e263-4b3e-99da-61db41eea694.png)
